src/test/java/StringTest.java

```diff
@@ -0,0 +1,54 @@
+import com.sun.swing.internal.plaf.synth.resources.synth_sv;
+import org.assertj.core.internal.bytebuddy.implementation.bind.MethodDelegationBinder;
+import org.assertj.core.util.VisibleForTesting;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+    @DisplayName("1,2를 문자열로 입력받아 spilt로 분리하면면 1과 2 문자열로 분리된다")
+    @Test
+    void splitTest1() {
+        assertThat("1,2".split(",")).containsExactly("1","2");
+    }
+
+    @Test
+    void exceptionTest1() {
+        //assertThat("1,2".split(",")).containsExactly("1","2");
+        assertThatThrownBy(() -> {
+            new Integer(1/0);
+        }).isInstanceOf(ArithmeticException.class);
+    }
+    //요구사항2
+    @Test
+    void substringTest() {
+        String a = "(1,2)";
+
+        int target = a.indexOf("(");
+        int target2 = a.indexOf(")");
```

Comment:
이것도 이런 식으로 명명해보면 읽기에는 좋은 코드가 됩니다. (변수명이 길어도 상관없어요)
```suggestion
        int positionOfLeftBracket = a.indexOf("(");
        int positionOfRightBracket = a.indexOf(")");
```